osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -72,10 +73,9 @@ private void load(OsuConfigManager config, NotificationOverlay notificationOverl
             {
                 hasShownNotificationOnce = true;
 
-                notificationOverlay?.Post(new SimpleNotification
-                {
-                    Text = @"The score overlay is currently disabled. You can toggle this by pressing Shift+Tab."
-                });
+                notificationOverlay?.Post(new SimpleNotificationContainer(

```

Comment:
at very least, this should not be a thing.

NotificationOverlay should know about the mapping between models and drawables, so we should just be posting the `new Notification()` and there should be type checks (switch in c#7) to create the drawable versions.